migrations/anser_test.go

```diff
@@ -11,9 +11,9 @@ import (
 
 func TestAnserBasicPlaceholder(t *testing.T) {
 	assert := assert.New(t)
-	app, err := Application(&mock.Environment{}, "mci_test")
+	app, err := Application(mock.NewEnvironment(), "mci_test")
 	assert.NoError(err)
-	assert.Len(app.Generators, 0)
+	assert.Len(app.Generators, 2)
```

Comment:
I think this assertion, and needing to update this test every time that we add a generator. Maybe just remove this assertion?

Reply:
Removed